filesystems/dropbox/src/main/kotlin/io/wavebeans/fs/dropbox/DropboxWbFileOutputStream.kt

```diff
@@ -0,0 +1,56 @@
+package io.wavebeans.fs.dropbox
+
+import com.dropbox.core.v2.DbxClientV2
+import com.dropbox.core.v2.fileproperties.PropertyGroup
+import com.dropbox.core.v2.files.CommitInfo
+import com.dropbox.core.v2.files.UploadSessionCursor
+import com.dropbox.core.v2.files.WriteMode
+import io.wavebeans.fs.core.WbFileOutputStream
+import java.io.ByteArrayInputStream
+import java.util.*
+
+class DropboxWbFileOutputStream(val client: DbxClientV2, val file: DropboxWbFile) : WbFileOutputStream() {
+
+    private val session = client.files().uploadSessionStart(false).finish()
+    private val buffer = ByteArray(65536)
+    private var count = 0
+    private var offset = 0L
+
+    override fun write(b: Int) {
+        buffer[count] = b.toByte()
+        if (count + 1 == buffer.size) {
+            flush()
+        }
+        count++
```

Comment:
after flush count is always 1, seems incorrect. Need to cover 65536+ sized files in tests